perfmetrics/scripts/run_load_test_and_fetch_metrics.sh

```diff
@@ -16,3 +24,4 @@ then
 else
   python3 fetch_metrics.py fio-output.json --upload
 fi
+sudo unmount $MOUNT_POINT
```

Comment:
Please fix this - add new line. Here and all such places.

Reply:
Sure! Thanks!